manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-editthiscookie</id>
+    <version>1.0.0</version>
+    <title>EditThisCookie</title>
+    <authors>Francesco Capano</authors>
+    <owners>gep13, chocolatey</owners>
```

Comment:
Small one: chocolatey should be listed first according to [§1.2.4](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#124-add-chocolatey-among-owners)